parcels/particleset.py

```diff
@@ -538,7 +538,7 @@ def execute(
 
         # Set up pbar
         if output_file:
-            logger.info(f"Output files are stored in {output_file.store}.")
```

Comment:
This needs to be rethought to work with all zarr stores. Agreed we should print path if there is one thought

Reply:
Done in d1cb597eae187d8c8316e997001e5cfa682e10dc

Reply:
Even better! Now done in 5237ccce9262db4274509a9411b76363fc13eb17